packages/workbox-runtime-caching/CacheFirst.mjs

```diff
@@ -93,19 +92,28 @@ class CacheFirst {
 
       if (process.env.NODE_ENV !== 'production') {
         if (response) {
-          logger.log(`Got response from network.`);
+          logs.push(`Got response from network.`);
         } else {
-          logger.log(`Unable to get a response from the network.`);
+          logs.push(`Unable to get a response from the network.`);
         }
       }
     } else {
       if (process.env.NODE_ENV !== 'production') {
-        logger.log(`Found a cached response in the '${this._cacheName}' ` +
-          `cache.`);
+        logs.push(
+          `Found a cached response in the '${this._cacheName}' cache.`);
       }
     }
 
     if (process.env.NODE_ENV !== 'production') {
+      logger.groupCollapsed(
+        messages.strategyStart('CacheFirst', event));
+      for (let log of logs) {
+        if (Array.isArray(log)) {
```

Comment:
Is `log` ever going to be an array? I'm not seeing any instances where `logs.push()` is called with anything other than a single string.

Reply:
Removed.